centos/centos-8.x/common/install_utils.sh

```diff
@@ -31,15 +36,22 @@ yum install -y numactl \
     cmake \
     libnl3-devel \
     libarchive \
-    libsecret
+    libsecret \
+    jq \
+    perl
```

Comment:
Just curious what is perl needed for?

Reply:
to resolve dnf check i.e, when you run `dnf check` on the VM it currently is looking for perl and is throwing a bunch of errors

Reply:
Is it due to the default mpitests_openmpi that comes with the base image?

Reply:
Yes